### DISCUSSION TITLE
Error starting zigbee-herdsman

### DISCUSSION THREAD

**Discussion by yatsik77** [Koenkk/zigbee2mqtt#7793] [General]
After update I am not able to start zigbee2mqtt.

My configurrations:
`zigbee-herdsman:adapter:zStack:adapter Detected znp version 'zStack30x' ({"transportrev":2,"product":2,"majorrel":2,"minorrel":7,"maintrel":2,"revision":20190523}) `

`Starting Zigbee2MQTT version 1.19.1 (commit #9bd46932)`
`Starting zigbee-herdsman (0.13.111)`

Error log:

`zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - osalNvLength - {"id":33} +3s`          
`zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,2,33,19,33,0,17] +3s`
`zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,2,97,19,0,0,112] +3s`
`zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,2,97,19,0,0,112] +1ms`
`zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 2 - 3 - 1 - 19 - [0,0] - 112 +0ms`
`zigbee-herdsman:adapter:zStack:znp:SRSP <-- SYS - osalNvLength - {"length":0} +3s`
`zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms`

This message happened few times, than timeout error:
`Zigbee2MQTT:error 2021-06-14 18:09:21: Error: network commissioning failed - timed out waiting for nib to settle`

What NIB does mean?

**Comment by Koenkk** [comment]
What adapter are you using?

**Reply by yatsik77** [reply]
DIY adapter based on Texas Instruments CC2538 PCB.

**Reply by Koenkk** [reply]
Can you try reflashing the adapter?

**Reply by yatsik77** [reply]
Yes. It helps.
After reflashiung with last FirmWare works perfect.
Thanks.
